package.json

```diff
@@ -36,8 +36,10 @@
   },
   "dependencies": {
     "@mdx-js/react": "^2.3.0",
+    "@mui/icons-material": "^5.11.16",
```

Comment:
We are currently using `react-icons` for icons package, update the icons from mui to react-icons

Reply:
👍